pkg/datagatherer/k8s/fieldfilter_test.go

```diff
@@ -18,10 +18,10 @@ func TestSelect(t *testing.T) {
 			"metadata": map[string]interface{}{
 				"name":      "example",
 				"namespace": "example",
-				"annotations": map[string]string{
+				"annotations": map[string]interface{}{
```

Comment:
We can expect all maps to be of type `map[string]interface{}`, that is the behavior for Kubernetes objects unmarshalled in unstructured objects (eg. https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/unstructured/helpers.go#L259).